modules/ddl-and-loading/pages/functions/token/gsql_ts_to_epoch_signed.adoc

```diff
@@ -1 +1,54 @@
 = gsql_ts_to_epoch_seconds_signed()
+
+Converts a timestamp in canonical string format to Unix epoch time (seconds since `Jan 1, 1970`).
+It supports dates from `0001-01-01` to `9999-12-31`, returning negative epoch values for dates prior to 1970. Invalid or out-of-range dates (e.g., `Feb 30th`) return `defaultTimestamp`.
+
+We also have another timestamp function, xref:ddl-and-loading:functions/token/gsql_ts_to_epoch.adoc[gsql_ts_to_epoch_seconds()], which converts a timestamp to Unix epoch time (seconds since Jan 1st, 1970) and returns an unsigned integer (`UINT`). For backward compatibility behavior, consider xref:ddl-and-loading:functions/token/gsql_ts_to_epoch_legacy.adoc[gsql_ts_to_epoch_seconds_legacy()].
```

Comment:
```suggestion
We also have another timestamp function, xref:ddl-and-loading:functions/token/gsql_ts_to_epoch.adoc[gsql_ts_to_epoch_seconds()], which converts a timestamp to Unix epoch time (seconds since Jan 1st, 1970) and returns an unsigned integer (`UINT`). For backward compatibility behavior (3.10.3 or earlier), consider xref:ddl-and-loading:functions/token/gsql_ts_to_epoch_legacy.adoc[gsql_ts_to_epoch_seconds_legacy()].
```